dev/source/docs/style-guide.rst

```diff
@@ -356,6 +356,30 @@ underscores.
 
     class ap_compass { };
 
+Enums
+---------------
+
+Prefer enum classes over raw enums. Enums should be CapitalCase, and be singular.
+
+**Right:**
```

Comment:
```suggestion
**Correct:**
```

Reply:
Can you explain why you want this change? The rest of the style guide uses right/wrong wording, so my changes are consistent.

Reply:
My assumption was that Correct would feed through a translator more accurately. Right is an idiom where correct is more definitive. Disregard if it matches the pattern.

Reply:
It matches the existing pattern, feel free to submit a PR though. Makes sense.